unison-share-api/src/Unison/Server/Backend.hs

```diff
@@ -1164,6 +1164,8 @@ resolveRootBranchHashV2 codebase mayRoot = case mayRoot of
     resolveCausalHashV2 codebase (Just h)
 
 -- | Determines whether we include full cycles in the results, (e.g. if I search for `isEven`, will I find `isOdd` too?)
+-- This was once used for term components, but is no longer, because now 'update' Does The Right Thing when editing a
```

Comment:
```suggestion
-- This was once used for term components, but is no longer, because now 'update' discovers any newly created components when editing a
```

I'm definitely not going to remember what "the right thing" was when I look back on this in a few months haha.

Reply:
I think the suggested edit is less clear, to be honest. Basically, 'update' does what you expect: propagate the change to dependents. That's "The Right Thing". It doesn't necessarily have to do with discovering new cycles, which is also the right thing to do, but is not the reason for this Do/DontIncludeCycleMates data type.

Reply:
But perhaps this comment should say that "the right thing" is propagating changes to dependents 😛

Reply:
What is the the reason for the `IncludeCycles`/`DontIncludeCycles` now? Then update the comment to explain that :)

Reply:
I thought the comment already explains that, although I see it's not 100% explicit:

> Determines whether we include full cycles in the results, (e.g. if I search for `isEven`, will I find `isOdd` too?)
> This was once used for term components, but is no longer, because now 'update' Does The Right Thing when editing a
> member of a term component. Decl components, on the other hand... :grimace:

Reply:
I guess there were two suggestions here: one to change `"The Right Thing"` to `<something else>`, and I something like `"propagates changes to dependents (including component-mates, which are de-facto dependents)"` would be my vote.

And then another to explain why `IncludeCycles` exists at all - but I see the comment actually never explained that, so maybe it should. Or link to a GH issue.